keps/sig-node/3573-device-plugin/README.md

```diff
@@ -493,12 +614,195 @@ Follow protobuf guidelines on versioning:
   * Freeze the package name to `apis/device-plugin/v1alpha1`
   * Have kubelet and the Device Plugin negotiate versions if we do break the API
 
-# References
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: `DevicePlugins`
+  - Components depending on the feature gate: Device Manager
+
+###### Does enabling the feature change any default behavior?
+
+No, in order to tap into this feature device plugins have to be deployed on the cluster.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, using `DevicePlugins` feature gate this feature can be disabled.
+Note that disabling the feature gate requires kubelet restart for the changes
+to take effect. In case no device plugin daemonset pods or pods consuming
+devices are running on the node, disabling feature gate shouldn't cause any issue.
+
+If the feature gate is being disabled on a node where such pods are running,
+it is the responsibliity of the cluster admin to ensure that the node is
+appropriately drained.
+
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+No impact on running pods in the cluster.
+When enabled, this feature provides the ability to expose and consume custom
+devices in a Kubernetes cluster. These device plugins are typically deployed
+as daemonset. Refer to [Installation](#installation) section for more details.
+
+Even when this feature is enabled, cluster admin needs to provision device plugins
+(typically deployed as a daemonset) on the cluster which subsequently be requested
+by the users. 
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes, covered by node e2e tests:
+* https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/device_manager_test.go
+* https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/device_plugin_test.go
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+* A rollback to Kubernetes version <1.10 can impact running workloads as they require
+  API version to exactly match for Kubelet and Device Plugin.
+
+* A rollout can fail in case a bug is introduced either in device manager or device
+  plugin preventing already running pods from restarting or new pods to start.
+
+###### What specific metrics should inform a rollback?
+
+Not Applicable, metrics specific to rollback are not available.
```

Comment:
I think this is more like "which metrics informs you that, depending on their values, you should perform a rollback"?
IIUC & IOW, which metrics can we use to assess the health of the feature and the cluster?